app/models/course/virtual_classroom_settings.rb

```diff
@@ -27,6 +27,20 @@ def braincert_whiteboard_api_key=(value)
     @settings.braincert_whiteboard_api_key = value
   end
 
+  # Returns BrainCert Virtual Classroom server region
+  #
+  # @return [String] The custom or default title of virtual classrooms component
+  def braincert_server_region
+    @settings.braincert_server_region || 7 # 7 is code for Singapore
```

Comment:
I think in this case we can just leave it as is. Not very fond of moving default to service as it seems more coherent to store settings defaults in a single place.